### PR TITLE
Readme for models

### PR DESCRIPTION
**Pull request by mosesdaudu001**

### CODE REVIEW

**Comment by mosesdaudu001** [comment]
Is this template okay?
@juliagsy @Madjid-CH

File: ivy_models/alexnet/README.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    </a>
+    <a href="https://discord.gg/G4aR9Q7DTN">
+        <img class="dark-light" style="float: left; padding-right: 4px; padding-bottom: 4px;" src="https://img.shields.io/discord/799879767196958751?color=blue&label=%20&logo=discord&logoColor=white">
+    </a>
+    <br clear="all" />
+
+AlexNet
+===========
+
+AlexNet competed in the ImageNet Large Scale Visual Recognition Challenge on September 30, 2012.
```
Comment (juliagsy) [inline ivy_models/alexnet/README.rst:30]: maybe move the references link here (link at alexnet directly)

File: ivy_models/alexnet/README.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    import ivy
+    from ivy_models import alexnet
+    ivy.set_backend("torch")
+    model = alexnet()
+
+The pretrained AlexNet model is now ready to be used, and is compatible with any other PyTorch code.
+See `this demo <https://unify.ai/demos/examples_and_demos/alexnet_demo.html>`_ for more usage example.
+
+
+References
```
Comment (juliagsy) [inline ivy_models/alexnet/README.rst:60]: could we remove this section?
(not sure if this is available) do we have a citation like the ivy one you added? could you please add/create one in the citation section?

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hey Moses! Thanks for the great work!
Just a few small comments, please let me know when it's ready again, thanks!

File: ivy_models/alexnet/README.rst (hunk 1 above)
Comment (mosesdaudu001) [reply ivy_models/alexnet/README.rst]: Done

File: ivy_models/alexnet/README.rst (hunk 2 above)
Comment (mosesdaudu001) [reply ivy_models/alexnet/README.rst]: Done

**Review by juliagsy — APPROVED** [review]
lgtm!

**Pull request merged**